src/pages/host/getHostEventString.tsx

```diff
@@ -319,7 +319,7 @@ export const HostEventLog: React.FC<{
   <span data-cy="host-event-logs">
     <StyledCollapse bordered={false}>
       <Panel header={title} key="1">
-        {{ isCode } ? (
+        {isCode ? (
```

Comment:
I think this could introduce a bug because of line 324 and made [this ticket](https://jira.mongodb.org/browse/EVG-13514) to address it.